test/signed-log.spec.js

```diff
@@ -262,5 +262,19 @@ apis.forEach((IPFS) => {
       assert.equal(log1.values.length, 1)
       assert.equal(log1.values[0].payload, 'one')
     })
+
+    it('throws an error if log is signed but the signature is valid and doesn\'t verify', async () => {
```

Comment:
Thank you for adding the test for this! ❤️ 

The description here says "throws an error..." but the assertions are not catching the error. I believe we need to:
```javascript
try {
// appends
// modify sig
// join
} catch (e) {
  err = e
}

assert.equal(err, `Error: Invalid signature in entry '${entry.hash}'`
```

Can you update the test to catch the error and verify it's thrown?